web/bitwarden/bitwarden.go

```diff
@@ -518,6 +518,7 @@ func Routes(router *echo.Group) {
 	ciphers.POST("", CreateCipher)
 	ciphers.POST("/create", CreateSharedCipher)
 	ciphers.GET("/:id", GetCipher)
+	ciphers.GET("/:id/details", GetCipher)
```

Comment:
Can you also update https://github.com/cozy/cozy-stack/blob/master/docs/bitwarden.md please?